lib/mixins/lambdas.js

```diff
@@ -5,6 +5,12 @@ module.exports = function lambdas(req, res) {
     renderField() {
       return function renderFieldMixin() {
         const mixin = this.mixin;
+        if (this.useWhen) {
+          if (req.sessionModel.get(this.useWhen.field) !== this.useWhen.value) {
```

Comment:
I mean if no value is given, then we assume that the value to test against is `true`
So this would assume the value is `true`

```
'field-name': {
  mixin: 'input-text',
  useWhen: {
    field: 'another-field'
  }
}
```